game-backend/src/helper/gameObjects.ts

```diff
@@ -18,3 +23,41 @@ export const countUnits = (state: State): number =>
 
 export const countFood = (state: State): number =>
   countGameObjectsPerType(state, GAME_OBJECT_TYPES.FOOD);
+
+export const getFreeFields = (state: State) => {
```

Comment:
Is there any correlation between `getGameObjectsPerType` and `getFreeFields`? I'd suggest creating a separate helper. Potentially changing the naming structure for these helpers.